doc/vim-go.txt

```diff
@@ -2197,6 +2197,12 @@ By default new terminals are opened in a vertical split. To change it
  let g:go_term_mode = "split"
 >
 
+Quickfix/location list takes up a lot of space~
+
+Sometimes the filename can take up a lot of space in the quickfix/location
+list windows. See https://github.com/fatih/vim-go/issues/1499 for some
+solutions to that.
+
```

Comment:
I don't think linking to a issue is helpful.If we add something here, it should contain the information so the user shouldn't look up and read something else again. Even pointing to a vim help page would make it better.

Reply:
Actually I was hesitant to add it to the FAQ in the first place as it's not really that "frequently asked". In general I'd say that if it's worth putting code in the FAQ, then it's probably also worth adding a proper vim-go feature.

In this particular case, there are many things you can do with Vim's quickfix window to make the experience better (hence plugins like vim-qf), but I don't think we need to add an FAQ entry for every one of them.

So maybe better to just close this?

Reply:
+1 for closing then.